app/controllers/languages_controller.rb

```diff
@@ -12,16 +13,27 @@ def show
   end
 
   def create
-    language = Language.create!(permit_params(:name, :code, :direction))
+    language = Language.create!(permit_params(:name, :code, :direction, :force_language_name))
     response.headers["Location"] = "languages/#{language.id}"
     render json: language, status: :created
   rescue ActiveRecord::RecordNotUnique
     raise Error::BadRequestError, "Code #{data_attrs[:code]} already exists."
   end
 
+  def update
+    language = load_language
+    language.update!(permit_params(:name, :direction, :force_language_name))
```

Comment:
Nothing here is checking the value of `force_language_name`; is that solely for the use of the client?

Reply:
I believe so, yes. That's what I read from https://jira.cru.org/browse/GT-2317 and Daniel has looked over this PR as well and surely would have commented something if not.

Reply:
yes, this is a flag to indicate to the mobile app that we should use the specified language name instead of relying on the device to localize the actual locale name